soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3355,21 +3399,13 @@ void DrawRandoEditor(bool& open) {
                     PaddedSeparator();
 
                     // Random Ganon's Trials
-                    /*
-                    ImGui::Text("Random Ganon's Trials");
-                    InsertHelpHoverText("Sets a random number or required trials to enter\nGanon's Tower.");
-                    SohImGui::EnhancementCombobox("gRandomizeGanonTrial", randoGanonsTrial, 2, 0);
+                    SohImGui::EnhancementCheckbox("Random Ganon's Trials", "gRandomizeGanonTrial");
+                    InsertHelpHoverText("Sets a random number or required trials to enter Ganon's Tower.");
                     if (CVar_GetS32("gRandomizeGanonTrial", 0) == 0) {
-                        ImGui::PopItemWidth();
                         SohImGui::EnhancementSliderInt("Ganon's Trial Count: %d", "##RandoTrialCount",
                                                        "gRandomizeGanonTrialCount", 0, 6, "", 6);
```

Comment:
not the biggest fan of having to set a slider to zero here, but i don't have any good ideas for the UI

only thing that comes to mind is a dropdown with
* Skip Trials
* Set # of Trials
* Random # of Trials

also might be good to add info to the tooltip explaining that certain trials will be required, not just any 3 if you pick 3

Reply:
Dropdown with slider only showing when "set number of trials" is enabled sounds good to me. And then have the slider go from 1 to 6. Would basically make it act like rainbow bridge setting.

Reply:
Addressed